spec/views/hyrax/base/show.html.erb_spec.rb

```diff
@@ -7,6 +7,7 @@
                      has_model_ssim: ['GenericWork'],
                      depositor_tesim: depositor.user_key,
                      description_tesim: ['Lorem ipsum lorem ipsum.'],
+                     abstract_tesim: ['Abstract specifics.'],
```

Comment:
Should there be a matching expectation in this file?

Reply:
abstract is not part of the head, so I actually don't think this is needed.